fastlane/jetpack_metadata/ar-SA/subtitle.txt

```diff
@@ -0,0 +1 @@
+اجعل موقعك على ووردبريس أسرع وأكثر أمانًا.
```

Comment:
Our automation regenerated all of the subtitles for Jetpack.

@oguzkocer pinged me about them in the [18.5 finalization PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/17392) but I missed it 😳 

> @mokagio For Jetpack subtitle fields, there were some languages with more than 30 characters. I've discussed this with @jkmassel and decided to remove the subtitle.txt files for this release in 2a73a0b. It looks like the subtitle.txt files didn't seem to have changed prior to this, so either this issue was already present in the previous releases, or we started to upload the subtitle field in this release. Either way, could you please look into this for a proper fix for next release?

The subtitles being too long is an issue I raised a while ago but utterly failed at following up on. I've simply been deleting them locally and filing it as a "I'll fix it in the next release" issue 😳 

Maybe this is the public shaming I need to actually get onto it 😅

Reply:
Ref: pcdRpT-rI